_posts/2020-10-22-how-to-update-to-aws-provider-v3.adoc

```diff
@@ -0,0 +1,189 @@
+---
+title: How to update Gruntwork references to Version 3 of the Terraform AWS provider
+categories: Foundations
+image: /assets/img/guides/infrastructure-as-code-library/grunty-grid.png
```

Comment:
I'm not sure what is the best icon to fit here. Used grunty for now because this is very specific to the Gruntwork products, but maybe there is something better to use?

Reply:
Perhaps a 3.0 icon? Or refresh icon? Or update badge icon?